tests/plugin_tests/test_link_announcer.py

```diff
@@ -142,3 +143,41 @@ def test_link_announce(match, test_str, res):
             mck.assert_called_with("Title: \x02" + res + "\x02")
         else:
             mck.assert_not_called()
+
+
+def test_link_announce_404():
+    url = 'http://example.com'
+    with RequestsMock() as reqs:
+        reqs.add(reqs.GET, url, status=404)
+
+        match = url_re.search(url)
+        assert match
+        mck = MagicMock()
+
+        assert print_url_title(match=match, message=mck) is None
+
+        mck.assert_not_called()
+
+
+@pytest.mark.parametrize('body,encoding', [
```

Comment:
This should have at least one negative check

Reply:
How would you implement a negative check in this case?

Reply:
Have one no encoding